_wiki/FAQ/LinuxAppSpecific.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Linux application-specific FAQ
+---
+
+### osu!stable
+
+#### osu! is not detecting input from my tablet (Wayland)
+
+Do not use `Absolute Mode` as your output mode. Use `Artist Mode` instead. Wine and XWayland currently has issues with parsing absolute mouse input.
```

Comment:
consider artist mode input smoothing before casually adding this, see OpenTabletDriver/OpenTabletDriver#2895

thoughts from others? Can also fast track a fix disabling the smoothing but it would require a new OTD release.